webapp/components/Group/GroupForm.vue

```diff
@@ -7,133 +7,184 @@
-          model="actionRadius"
-          :options="['regional', 'national', 'continental', 'global']"
-          icon="globe"
-          :placeholder="`${$t('group.actionRadius')} …`"
-        ></ds-select> -->
-        <ds-text class="select-label">
-          {{ $t('group.actionRadius') }}
-        </ds-text>
-        <ds-icon class="select-icon" name="globe" />
-        <select
-          class="select"
-          :options="actionRadiusOptions"
-          :value="formData.actionRadius"
-          @change="changeActionRadius($event)"
-        >
-          <option
-            v-for="actionRadius in actionRadiusOptions"
-            :key="actionRadius"
-            :value="actionRadius"
-          >
-            {{ $t(`group.actionRadii.${actionRadius}`) }}
-          </option>
-        </select>
-        <!-- location -->
-        <ds-select
-          id="city"
-          :label="$t('settings.data.labelCity')"
-          v-model="formData.locationName"
-          :options="cities"
-          icon="map-marker"
-          :placeholder="$t('settings.data.labelCity') + ' …'"
-          :loading="loadingGeo"
-          @input.native="handleCityInput"
-        />
-        <!-- TODO: implement clear button -->
-        <!-- <base-button icon="close" circle ghost size="small" :disabled="formData.locationName.length === 0" @click="clear" /> -->
-      </ds-space>
-      <ds-space margin-top="large">
-        <categories-select
-          v-if="categoriesActive"
-          model="categoryIds"
-          :existingCategoryIds="formData.categoryIds"
-        />
-      </ds-space>
-      <ds-space margin-top="large">
-        <nuxt-link to="/my-groups">
-          <ds-button>{{ $t('actions.cancel') }}</ds-button>
-        </nuxt-link>
-        <ds-button
-          type="submit"
-          icon="save"
-          :disabled="update ? submitDisableEdit : submitDisable"
-          primary
-          @click.prevent="submit()"
-        >
-          {{ update ? $t('group.update') : $t('group.save') }}
-        </ds-button>
-      </ds-space>
+      <template #default="{ errors }">
+        <base-card>
+          <!-- Group Name -->
+          <ds-input
+            :label="$t('group.name')"
+            model="name"
+            name="name"
+            autofocus
+            :placeholder="`${$t('group.name')} …`"
+          />
+          <ds-text align="right">
+            <ds-chip size="base" :color="errors && errors.name && 'danger'">
+              {{ `${formData.name.length} / ${formSchema.name.max}` }}
+              <base-icon v-if="errors && errors.name" name="warning" />
+            </ds-chip>
+          </ds-text>
+
+          <!-- Group Slug -->
+          <ds-space margin-top="large">
+            <ds-input
+              v-if="update"
+              :label="$t('group.labelSlug')"
+              model="slug"
+              icon="at"
+              :placeholder="`${$t('group.labelSlug')} …`"
+            ></ds-input>
+          </ds-space>
```

Comment:
This does not work like expected if we edit in my eyes.
We have to talk about how it should wokr with @Mogge I guess and it has as well to do with the backend behaviour.

Will we do it in another separate issue?

Reply:
yes, let's make this a separate issue

Reply:
Here it is already #5397